main/settings.py

```diff
@@ -453,108 +450,11 @@
     description="How long the robots.txt file should be cached",
 )
 
-# Social Auth Configuration
-
 AUTHENTICATION_BACKENDS = (
-    "authentication.backends.apisix_remote_user_org.ApisixRemoteUserOrgBackend",
```

Comment:
need this one.. fwiw this isn't social-auth, it's just here because it seemed like a good place to put it.